client/index.html.tpl

```diff
@@ -47,8 +47,8 @@
 			<article id="windows">
 				<div id="loading" class="window active">
 					<div id="loading-status-container">
-						<img src="img/logo-vertical-transparent-bg.svg" class="loading-logo-bright" alt="The Lounge" width="256" height="170">
-						<img src="img/logo-vertical-transparent-dark.svg" class="loading-logo-dark" alt="The Lounge" width="256" height="170">
+						<img src="img/logo-vertical-transparent-bg.svg" class="logo" alt="The Lounge logo" width="256" height="170">
```

Comment:
There's no reason for the `alt` to say `logo`.

Reply:
It's not clear, because screen readers read this as is, so `alt` should describe what is in the image. As it is, a screen reader just says "The Lounge, image" and a person with visual impairment could not tell what that means: is it a screenshot of The Lounge? Is it a lounge chair lol?

That said, I couldn't find a definitive truth online, even on accessibility-related website: WebAIM doesn't specify "logo" in their logo alt, WCAG has "W3C logo", etc.
So I'll stick without for now.